provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+docker build -t al2-graalvm .
+sam build --use-container --build-image al2-graalvm
```

Comment:
I wonder if we can move;
- `docker build -t al2-graalvm .` into `Makefile`
- And set `build_image` and `use_container` property in samconfig.toml file as default values

So that users can just run `sam build` when they initialized this project.

Reply:
Well that is my bad I missed we are building inside the container :)

How about keeping the `docker build -t al2-graalvm .` inside another `.sh` file and mention that in the readme that users needs to run that first. And afterwards, they can just run `sam build` with default parameter options?